docs/tutorials/getting_started.rst

```diff
@@ -2,46 +2,89 @@
-           [ -38.-38.j,  -40. +0.j],
-           [-105.+60.j,   85.-15.j]], dtype=complex64)
-    >>> fh.close()
-
-For other file formats, a bit more information is needed.  Below, we cover the
-basics of :ref:`inspecting files <getting_started_inspecting>`, :ref:`reading
-<getting_started_reading>` from and :ref:`writing <getting_started_writing>`
-to files, and :ref:`converting <getting_started_converting>` from one format
-to another.  We assume that Baseband as well as `NumPy
-<http://www.numpy.org/>`_ and the `Astropy`_ units module have been imported::
+To install Baseband, please see :ref:`Installing Baseband <install_baseband>`.
+We recommend installing using `pip <http://www.pip-installer.org/en/latest/>`_
+within a `virtual environment <https://virtualenv.pypa.io/en/stable/userguide/>`_.
+
+.. _getting_started_opening:
+
+Opening Files
+=============
+
+When using Baseband, we typically will also use `numpy` and the `astropy.units`
+and `astropy.time` modules (these should have been installed with Baseband if
+they weren't already available). Let's import all of these::
 
     >>> import baseband
     >>> import numpy as np
     >>> import astropy.units as u
+    >>> import astropy.time as t
+
+For this tutorial, we'll use two sample files::
+
+    >>> from baseband.data import SAMPLE_VDIF, SAMPLE_MARK5B
+
+The first file is a VDIF one created from `EVN <http://www.evlbi.org/>`_/`VLBA
+<https://public.nrao.edu/telescopes/vlba/>`_ observations of `Black Widow
+pulsar PSR B1957+20 <https://en.wikipedia.org/wiki/Black_Widow_Pulsar>`_,
+while the second is a Mark 5B from EVN/`WSRT
+<https://www.astron.nl/radio-observatory/public/public-0>`_ observations of the
+same pulsar.
+
+To open the VDIF file::
+
+    >>> fh_vdif = baseband.open(SAMPLE_VDIF)
+
+Opening the Mark 5B file is slightly more involved, as not all required
+metadata is stored in the file itself::
+
+    >>> fh_m5b = baseband.open(SAMPLE_MARK5B, nchan=8, sample_rate=32*u.MHz,
+    ...                        ref_time=t.Time('2014-06-13 12:00:00'))
+
+Here, we've manually passed in as keywords the number of |channels|, the
+:term:`sample rate` (number of samples per channel per second) as an
+`astropy.units.Quantity`, and a reference time within 500 days of the start of
+the observation as an `astropy.time.Time`.  That last keyword is needed to
+properly read timestamps from the Mark 5B file.
+
+`baseband.open` tries to open files using all available formats, returning
+whichever is successful.  If you know the format of your file, you can pass
+it with the ``format`` keyword, or directly use the format opener (for VDIF, it
+is `baseband.vdif.open`).
+
+Do you have a sequence of files you want to read in?  You can pass a list of
+filenames to `baseband.open`, and it will open them up as if they were a single
+file!  See :ref:`Reading or Writing to a Sequence of Files
+<using_baseband_multifile>` for more information.
 
-.. _getting_started_inspecting:
+.. note::
```

Comment:
I think this is a distraction here; if you want to keep it, perhaps instead have a footnote on how `baseband.open` tries all formats above (but really, that has a help; don't aim for completeness here).